testutils/wallet.go

```diff
@@ -0,0 +1,5 @@
+package testutils
+
+func CreateWallet(){
+
```

Comment:
why 2 empty tests?

Reply:
Testutils is temporarily on the 0.10-dev branch, but in order not to let Matser report an error, there is only an empty test.